app/Http/Resources/V1/ProjectDetailResource.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Resources\V1;
+
+use App\Http\Resources\V1\ProjectUserResource;
```

Comment:
We are not using ProjectUserResource, right? If not, take it out.

Reply:
check it